fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/FabricModResourcePack.java

```diff
@@ -47,7 +47,7 @@ public InputSupplier<InputStream> openRoot(String... pathSegments) {
 		String fileName = String.join("/", pathSegments);
 
 		if ("pack.mcmeta".equals(fileName)) {
-			String description = "Mod resources.";
+			String description = Text.translatable("pack.fabricapi.description");
```

Comment:
This does not compile as `Text` is not compatible with `String`.

Reply:
Yep, the proper replacement is to use `{"translate":"the tl key here"}` instead of `"description text"` in the json

Reply:
This should work:
```suggestion
			String description = Text.translatable("pack.fabricapi.description").getString();
```
I also understand what @Juuxel meant, but not _where_ would that apply? In other words, I don't see a relevant pack.mcmeta in the repo where I could directly apply it to.

Reply:
@Madis0 Make sure that works for data packs on a dedicated server. It should be fine in theory, especially with dedicated servers loading modded translations, but ideally we should use @Juuxel's strategy if that works.

Reply:
> but not where would that apply

The very next line in that file